exp/views/study.py

```diff
@@ -194,16 +194,22 @@ def get_form_kwargs(self):
     # UserPassesTestMixin.get_test_func()
     test_func = user_can_edit_study
 
+    # Rely on form validation to make sure structure is valid JSON; don't load/dump so we
+    # preserve formatting and ordering.
+    #
```

Comment:
get rid of empty comment line

Reply:
sorry about that! edited this comment which was actually a relic of trying a variety of approaches to smoothly move to checking for valid JSON but storing the actual string so that from the user's standpoint it's like a big text box.